client/lib/plans/features-list.js

```diff
@@ -128,7 +128,7 @@ export const FEATURES_LIST = {
 
 	[ constants.FEATURE_FREE_DOMAIN ]: {
 		getSlug: () => constants.FEATURE_FREE_DOMAIN,
-		getTitle: () => i18n.translate( 'Free Domain for One Year' ),
+		getTitle: () => i18n.translate( 'Free domain for one year' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **29** times:
`translate( 'Free domain for one year' )` *ES Score: 12*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/card-headings-form-labels-sentence-case)


:information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?